couler/core/run_templates.py

```diff
@@ -37,59 +37,148 @@ def run_script(
     image,
     command=None,
     source=None,
+    args=None,
+    output=None,
+    input=None,
     env=None,
-    resources=None,
     secret=None,
+    resources=None,
     timeout=None,
     retry=None,
     step_name=None,
     image_pull_policy=None,
     pool=None,
+    enable_ulogfs=True,
     daemon=False,
+    volume_mounts=None,
+    working_dir=None,
+    node_selector=None,
 ):
-    """Generate an Argo script template.  For example,
+    """
+    Generate an Argo script template.  For example,
     https://github.com/argoproj/argo/tree/master/examples#scripts--results.
-    Step_name is only used for annotating step while developing step zoo.
+    :param image:
+    :param command:
+    :param source:
+    :param args:
```

Comment:
Would you like to help us fill in these docstrings? Except the last three args that are only used internally and will be removed.

Reply:
done